### ISSUE TITLE
Design/Style attachment nuggets

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#47]
# Manta Ray and BGS Done criteria
- [x] Style nugget
- [x] ISSUE there's going to be an image added to this. (see issue #44) NOTE the client has selected the version with the SMALLER IMAGE
- [x] **ISSUE** Ensure all the text aligns with the same margin (including doc type and file size)
- [x] **ISSUE** If there's no cover the text should align left (as per the design)
- [x] **ISSUE** move the cover image to the right hand side of the text

<img width="283" alt="33845142-71941bba-de70-11e7-9cd5-7098a3f228a6" src="https://user-images.githubusercontent.com/3796718/34947258-9dfd38f8-f9d7-11e7-9c62-4bfa8d3c4d92.png">

<img width="283" alt="screen shot 2017-12-04 at 15 20 36" src="https://user-images.githubusercontent.com/3796718/33574342-bafb4540-d906-11e7-87d8-2f94d4652d85.png">

# Designs and Drupal pages
- Appears on [Our Purpose page design](https://invis.io/4EDQ458R5#/266149708_Our_Purpose)
- [Right hand column](http://bgs-dev.mrmdev.co.uk/examples/right-hand-column)

**Comment by angievanhegan** [comment]
Boxes on RHS
Is there a limit to how many options/links that we could add within the box? Eg could we have 3 links for related content and 6 attachments listed. If not - should we set a limit to maintain the integrity of the design?
Can the boxes:

contain more than 2-3 lines of text per link?
have images within them (eg is the attachment is a printed document, we would have the document cover art that we could add here)
in the related content box, could we specify what type of document it is (blog, news, resource etc)

Are these boxes set to the top right of the page? And what is the impact visually of increasing their length if we add lots of text to them?

**Comment by angievanhegan** [comment]
@pipe-manta This is unblocked now as the client has selected one from the options that Cat gave her. It's ready to work on when you are.

**Comment by meldera** [comment]
@angievanhegan @russ This isn't "adding an image", but completely rebuilding how attachments work. OK to proceed?..

**Comment by angievanhegan** [comment]
@russ-manta Can you let us know how much work this is please? Thanks

**Comment by russ-manta** [comment]
@angievanhegan what's the scope here? Is it that they can add their own custom image to the attachment or is this an image of the first page of the attachment?

**Comment by angievanhegan** [comment]
@russ-manta sorry, it'll be the first page of the attachment. Thanks

**Comment by russ-manta** [comment]
@angievanhegan either way Arseni thinks 1/2 day for this

**Comment by russ-manta** [comment]
@angievanhegan @ican-icant  Arseni can work on this today if we're happy to procees

**Comment by ican-icant** [comment]
@meldera @russ-manta Please proceed

**Comment by meldera** [comment]
http://bgs-dev.mrmdev.co.uk/attachments

**Comment by angievanhegan** [comment]
@russ-manta all the text (doc title and doc type) needs to align with the same vertical margin. The file type is currently wrapping under the image.  Thanks

<img width="297" alt="screen shot 2017-12-01 at 17 00 22" src="https://user-images.githubusercontent.com/3796718/33505407-29262af8-d6b9-11e7-9563-6067d3edd561.png">

**Comment by angievanhegan** [comment]
@StephMSlack This is ready for you to sign off. Note that it doesn't include the Related Content nugget. That's on a separate issue.

**Comment by StephMSlack** [comment]
@angievanhegan Recia had commented this morning that the text for the headings on all the right hand column nuggets seems very large. I'm waiting for her to review the updated page so will confirm when I have heard back from her

**Comment by angievanhegan** [comment]
@StephMSlack Bear in mind that they appear much bigger on the smaller screen. When you see them on a bigger screen they don't look big at all. I'd just want to make sure we're not penalising people with big screens!

**Comment by StephMSlack** [comment]
@angievanhegan please can we reduce the size of the text - it feels very large compared to the rest of the body text. Also see my comment on #65 re: changing the text colour to pink

Also a query from Recia - would it be a big job to have the images sitting to the right of the attachment title (v's the current option of to the left)

**Comment by angievanhegan** [comment]
@cat-manta please can we try putting the cover image on the right hand side? It'd be good just to see how this looks in the design. Thank you

**Comment by angievanhegan** [comment]
@StephMSlack I've incorporated the rh nugget text size issue into #89 so that that issue covers all rh nuggets

**Comment by cat-manta** [comment]
@StephMSlack I have moved the attachment image to the right of the text if you could see which you prefer:
https://invis.io/4EDQ458R5#/266149708_Our_Purpose

**Comment by angievanhegan** [comment]
@cat-manta That cover looks really tiny...

**Comment by StephMSlack** [comment]
Thanks @cat-manta and @angievanhegan I'm keeping Recia in the loop as she was the one who had asked about the change.

**Comment by cat-manta** [comment]
@StephMSlack Just catching up with issues from before Christmas, this one wasn't properly passed over because of the size, but we think it's the title that makes it look a bit odd and small, so have added in an extra word which should help.

**Comment by StephMSlack** [comment]
@cat-manta Apologies for the delay. Just checking with Recia who requested the change. Will get back to you asap. Steph

**Comment by StephMSlack** [comment]
@angievanhegan @cat-manta image to the right is fine and the preferred option. Thanks, Steph

**Comment by angievanhegan** [comment]
@russ-manta @pipe-manta This one is ready for build.

**Comment by angievanhegan** [comment]
@StephMSlack This last issue is ready for you to sign off (see the list at the top - the issue is to put the cover for the download on the right hand side). Thanks

**Comment by StephMSlack** [comment]
@angievanhegan this is fine
